config/toggles.js

```diff
@@ -69,6 +69,10 @@ const longTermToggles = {
   enforceWritePermission: environmentVariableTrue(pe,
                                         'ENFORCE_WRITE_PERMISSION'),
 
+  // Enable api activity logging
+  enableApiLogs: environmentVariableTrue(pe,
```

Comment:
Can we use "ENABLE_API_ACTIVITY_LOGS" so it follows the same naming convention as the worker one?